tutorials/graph-hello-world/graph-hello-world.md

```diff
@@ -0,0 +1,166 @@
+        const options = {
+            method: "get",
+            headers:{
+                "Accept": "application/json",
+                "apiKey": "your APIkey"
+            }
+        };
+        const response = await fetch(url, options);
+        console.log(`${response.status} (${response.statusText})`) // for debugging
+        const json = await response.json();
+        return json;
+    }
+}
+
+module.exports = Graph;
+
+```
+Note that we hard-coded the SAP Graph server that uses SAP API Business Hub sandbox. This allows us to focus here on the data access API, without requiring you to first configure your own SAP Graph business data graph and deal with all the complex aspects of security and authentication, which will be the subject of future tutorial parts.
+
+Since we are accessing the sandbox landscape data via SAP API Business Hub, you will need to insert your API Key (a short string) into the code above. Where do you get this key? Go and login to <https://api.sap.com/settings> and click on **Show API Key** to see and save it.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 5: ](Write a Simple Application)]
+
+Now we are ready to write our first, simple server-side application that uses SAP Graph. Paste the following into a new file, called `hellograph.js`, in the `src` folder:
+
+```
+const express = require("express");
+const app = express();
+const Graph = require("./graph");
+const port = 3004;
+
+const graph = new Graph();
+
+app.get('/sap*', async (req, res) => {
+    const response = await graph.get(req, `${req.url}`, "");
+    res.send(`<pre><code>${JSON.stringify(response, null, 2)}</code></pre>`);
+});
+
+app.listen(port, () => { console.log(`Explore SAP Graph at http://localhost:${port}`) });
+
+```
+As mentioned, our code uses a popular Node.js package called **Express**. On line 8, we define the action taken when our server-side app is called from a browser. Here, we simply take the URL that we received (`req.url`), and pass it through, as-is, to SAP Graph. We then show the returned data from SAP Graph as a raw result in the browser screen.
+
+Our server-side app is ready. To run it, use your terminal console to enter:
+
+```
+node hellograph.js
+```
+The console will show:
+
+> _Explore SAP Graph at <http://localhost:3004>_
+
+Now the fun begins. The application expects a well-formed query URL to function properly. Open a browser window or tab, and enter the following query URL:
+
+```
+http://localhost:3004/sap.graph/SalesQuote?$top=2
+```
+The browser will invoke your application code, which will call SAP Graph to fetch the data, and if all went well, you will see the following output on your screen:
+
+!![Output](blog2_output_sm.png)
+
+This output should look very similar to what you saw in the SAP Graph Navigator for the `SalesQuote` entity in Step 1.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 6: ](Use SAP Graph Securely with Real Data and Authentication)]
+
```

Comment:
Please remove this line!

Reply:
Not entirely clear what is meant here, but I removed the empty line preceding the step-begin.